README.md

```diff
@@ -23,6 +23,7 @@ A searchable sort-able and group-able version of the list of companies below is
 ## D - F
 - [Deputy](https://www.deputy.com/au/careers) | Sydney CBD, Sydney |  PHP, Javascript, Android, SQL
 - [Domain Group](https://domaingroup.csod.com/ux/ats/careersite/4/home?c=domaingroup) | Sydney CBD, Sydney | PHP, Javascript, Nodejs, React, Android, Python, C#, Swift
+- [Easygo](https://easygo.io/careers/) | Melbourne CBD, Melbourne | JavaScript, NodeJs, React
```

Comment:
Sorry, but look like the site is down though.

Reply:
> Sorry, but looks like the site is down.

I was able to open it fine, how did you try to access it?

Reply:
Looks fine now. How did you know that they sponsor work visa?